pnpm-workspace.yaml

```diff
@@ -3,3 +3,29 @@ packages:
   - packages/@n8n/*
   - packages/@n8n_io/*
   - cypress
+
+catalog:
+  '@types/basic-auth': ^1.1.3
+  '@types/express': ^4.17.21
```

Comment:
Some of these are used only in the backend packages. Should we have a separate `backend` catalog like we have one for frontend?